historical_data_api_client/utils.py

```diff
@@ -0,0 +1,24 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from datetime import datetime
+DATE_TIME_FORMAT = "%Y-%m-%dT%H:%M"
+
+
+def validate_correct_datetime_format(time_str: str):
+    """Validate time string follows DATE_TIME_FORMAT."""
+    datetime.strptime(time_str, DATE_TIME_FORMAT)
```

Comment:
I'd suggest making this more flexible:
```suggestion
def validate_correct_datetime_format(time_str: str, time_format: str):
    """Validate that time string respects the provided time format."""
    datetime.strptime(time_str, time_format)
```
And then you can pass the `DATE_TIME_FORMAT` from the `cli.py` module. E.g.:
```python
DATE_TIME_FORMAT = "%Y-%m-%dT%H:%M"
# ...
utils.validate_correct_datetime_format(start_time, DATE_TIME_FORMAT)
utils.validate_correct_datetime_format(end_time, DATE_TIME_FORMAT)
```